### DISCUSSION TITLE
Voice in FSK direct mode

### DISCUSSION THREAD

**Discussion by n9wxu** [jgromes/RadioLib#552] [Ideas]
The FSK mode works by modulating the direct pin and letting the FSK filters keep the harmonics down.  Has anyone tried modulating audio (voice) signals on PWM and relying upon the FSK filters to produce a recognizable voice on the other end?

**Comment by jgromes** [comment]
At one point I tried something along those lines: https://www.youtube.com/watch?v=FAT3iFEyLvE

The source was an MP3 file as opposed to a microphone, but I think that should work as well.

**Comment by n9wxu** [comment]
perfect, just sorting out a suitable PWM for the SAMD21.
